src/compiler/ast-node-bop.bas

```diff
@@ -1539,7 +1539,7 @@ function astNewBOP _
 			return rtlMathLongintMOD( dtype, l, ldtype, r, rdtype )
 		end select
 
-	' Trap ANDALSO, ORELSE, handle floats, and convert to IIF
```

Comment:
Here, and in many other files, comments with single quote are stripped.  My preference is 2 single quotes to mark a comment, but that does not mean single quoted comments should be removed.